hw/ip/rbox/data/rbox.hjson

```diff
@@ -0,0 +1,576 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{ name: "rbox",
+  clock_primary: "clk_i",
+  other_clock_list: [
+    "clk_aon_i"
+  ]
+  bus_device: "tlul",
+  bus_host: "none",
+  reset_primary: "rst_ni",
+  other_reset_list: [
+    "rst_slow_ni"
+  ],
+  interrupt_list: [
+    { name: "rbox_intr_o",
+      desc: "combo or keyboard triggered interrupt",
+    }
+  ],
```

Comment:
similar to `dcd` and others, you will probably need `wakeup_list` and `reset_list` here long term. 
Feel free to have a look at `pinmux.hjson` or `aon_timer.hjson` for a preview.

Reply:
Okay. I understand the reset list. rbox has rst_ni and rst_slow_ni. So not on the wakeup list. rbox is on the "Always-on" domain. I have interrupts to wake up the CPU.

  name: "PINMUX",
  clock_primary: "clk_i",
  reset_primary: "rst_ni",
  other_clock_list: [
    "clk_aon_i",
  ],
  other_reset_list:
  [
    "rst_aon_ni"
  ],
  bus_device: "tlul",
  regwidth: "32",

  wakeup_list: [
    { name: "aon_wkup_req",
      desc: "pin wake request"
    }
  ],

Reply:
@esshiu I believe any wakeup request needs to go to the power manager via a `wakeup_list` signal (correct me if I am wrong, @tjaychen). The CPU cannot wake up automatically from deep sleep via an IRQ only.

Reply:
Andrey Pronin and I discussed the ULP wakeup case before. GSC FW runs on CPU. If CPU is powered down, an interrupt won't do any good. +apronin@opentitan.org